src/test/java/org/fundacionjala/sfdc/stepDefinition/CreateContactSteps.java

```diff
@@ -0,0 +1,72 @@
+package org.fundacionjala.sfdc.stepDefinition;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+/**
+ * CreateContactSteps.java.
+ * Class for steps createContact.feature.
+ */
+public class CreateContactSteps {
+
+
+    /**
+     * Given step.
+     */
+    @Given("^I am going to app launcher page$")
+    public void iAmGoingToAppLauncherPage() {
+        System.out.println("I go to app launcher button.");
+
+
+    }
+
+    /**
+     * Given step.
+     */
+    @Given("^I am on the new contact page$")
+    public void iAmOnTheNewContactPage() {
+        System.out.println("This Step open the Firefox and go to login page of the application.");
+
+       /* contact = new CreateContacts(web);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)